src/style.css

```diff
@@ -6,9 +6,15 @@
   box-sizing: border-box;
 }
 
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: whitesmoke;
+}
+
 header {
   padding: 20px;
   font-size: 25px;
+  color: blue;
 }
 
```

Comment:
I also refactor this code to update Score Board if a new score is successfully posted. Please implement all this, and your project will be just perfect. 

Refactored code
```javascript
const updateBoard = (user) => {
  const li = `
  <li class="score--list--item">
  <span>${user.user}:</span>
  <span>${user.score}</span>
  </li>`;
  userContainer.insertAdjacentHTML('beforeend', li);
  userContainer.classList.remove('hide--score--list--border');
};

form.addEventListener('submit', async (e) => {
  e.preventDefault();
  
  const nameInput = form.querySelector('#name');
  const scoreInput = form.querySelector('#score');
  const user = nameInput.value;
  const score = scoreInput.value;
  const userInfo = { user, score };

  try {
    const res = await createGameOrPostScore(`${baseUrl}${gameId}/scores/`, userInfo);
    
    if (res?.result === 'Leaderboard score created correctly.') {
      updateBoard(userInfo);
    }
  } catch (error) {
    console.error('Error posting the score:', error);
  }

  nameInput.value = '';
  scoreInput.value = '';
});
```